integration-tests/tests/billing-projects.js

```diff
@@ -35,19 +34,19 @@ const setAjaxMockValues = async (testPage, ownedBillingProjectName, spendCost) =
     window.ajaxOverridesStore.set([
       {
         filter: { url: /api\/billing\/v2$/ },
-        fn: () => async () => { return new Response(JSON.stringify(projectListResult), { status: 200 }) }
+        fn: () => () => Promise.resolve(new Response(JSON.stringify(projectListResult), { status: 200 }))
```

Comment:
It complains about async? I thought I followed our usual pattern.

Reply:
It complained about `async` functions with no `await`.

Reply:
I see, thanks!